config/passport.js

```diff
@@ -76,7 +77,21 @@ module.exports = function(passport, models) {
                         .catch(function(err) {
                             return done(err);
                         });
-                })
+                    levelProgress.findOrCreate({where: {
+                        user_id: user.id
+                    }, defaults: {
+                        user_id: user.id,
+                    }})
+                        .spread(() => {
```

Comment:
Identation here..

Reply:
. spread can be started from previous line itself..

Reply:
Done.